src/propTypes.js

```diff
@@ -6,6 +6,15 @@ export default {
   volume: PropTypes.number,
   width: PropTypes.oneOfType([ PropTypes.string, PropTypes.number ]),
   height: PropTypes.oneOfType([ PropTypes.string, PropTypes.number ]),
+  soundcloud: PropTypes.shape({
```

Comment:
I'm thinking we should perhaps use `soundcloudConfig`, `youtubeConfig` and `vimeoConfig` for clarity.